client/views/room/contexts/RoomContext.ts

```diff
@@ -18,13 +25,16 @@ export const useRoom = (): IRoom => {
 	return room;
 };
 
-export const useOmnichannelRoom = (): IOmnichannelRoom => {
+export const useOmnichannelRoom = (): IOmnichannelRoom & IOmnichannelSubscription => {
```

Comment:
Why do we need to support IOmnichannelSubscription interface here given that useOmnichannelRoom always return just IRoom ?